python/GafferImageUI/ImageViewUI.py

```diff
@@ -1300,49 +1350,217 @@ def __toggle( self, *args ):
 
 		self.getPlug().setValue( not v )
 
-class _CompareCatalogueOutputPlugValueWidget( GafferUI.PlugValueWidget ) :
+class _CompareImageWidget( GafferUI.Frame ) :
 
-	def __init__( self, plug, **kw ) :
+	def __init__( self, plug ) :
 
-		self.__button = GafferUI.MenuButton(
-			image = "catalogueOutput1.png",
-			menu = GafferUI.Menu(
-				Gaffer.WeakMethod( self.__menuDefinition ),
-				title = "Catalogue Output",
-			)
+		GafferUI.Frame.__init__( self, borderWidth = 0, borderStyle = GafferUI.Frame.BorderStyle.None_ )
+
+		self._qtWidget().setFixedHeight( 15 )
+		self.__node = plug.node()
+		self.__scriptNode = plug.node()["in"].getInput().node().scriptNode()
+		self.__defaultNodeSet = Gaffer.StandardSet( [] )
+		self.__nodeSet = self.__defaultNodeSet
+		self.__catalogueOutput = 0
+
+		row = GafferUI.ListContainer( orientation = GafferUI.ListContainer.Orientation.Horizontal )
+		with row :
+
+			self.__bookmarkNumber = GafferUI.Label( horizontalAlignment=GafferUI.Label.HorizontalAlignment.Right )
+			self.__bookmarkNumber.buttonPressSignal().connect( Gaffer.WeakMethod( self.__showEditorFocusMenu ), scoped=False )
+
+			self.__icon = GafferUI.Button( hasFrame=False, highlightOnOver=False )
+			self.__icon._qtWidget().setFixedHeight( 13 )
+			self.__icon._qtWidget().setFixedWidth( 13 )
+			self.__icon.buttonPressSignal().connect( Gaffer.WeakMethod( self.__showEditorFocusMenu ), scoped=False )
+
+			# TODO - why does the pinning dropdown use collapsibleArrowDown instead of menuIndicator?
+			self.__menuButton = GafferUI.Button( image="menuIndicator.png", hasFrame=False, highlightOnOver=False )
+			self.__menuButton._qtWidget().setObjectName( "menuDownArrow" )
+			self.__menuButton.buttonPressSignal().connect( Gaffer.WeakMethod( self.__showEditorFocusMenu ), scoped=False )
+
+		self.addChild( row )
+
+		self.buttonPressSignal().connect( Gaffer.WeakMethod( self.__showEditorFocusMenu ), scoped=False )
+
+		self._setState( self.__defaultNodeSet, 1 )
+
+	def _setState( self, nodeSet, outputIndex ):
+		self.__nodeSet = nodeSet
+		self.__catalogueOutput = outputIndex
+		self.__memberAddedConnection = self.__nodeSet.memberAddedSignal().connect(
+			Gaffer.WeakMethod( self.update ), scoped = True
 		)
-		self.__button._qtWidget().setMaximumWidth( 25 )
+		self.__memberRemovedConnection = self.__nodeSet.memberRemovedSignal().connect(
+			Gaffer.WeakMethod( self.update ), scoped = True
+		)
+		self.update()
 
-		GafferUI.PlugValueWidget.__init__( self, self.__button, plug, **kw )
+	# TODO - hotkeys?
```

Comment:
I think we're probably fine to omit hotkeys for this.

Reply:
Removed comment.